src/components/Combobox/__stories__/OptionRenderer.jsx

```diff
@@ -3,7 +3,7 @@ import Avatar from "../../Avatar/Avatar";
 export const optionRenderer = ({ src, id, label, position }) => {
   return (
     <div className="person-picker-mock-option">
-      <Avatar size={Avatar.sizes.SMALL} src={src} type={Avatar.types.IMG} key={id} />
+      <Avatar customSize={20} src={src} type={Avatar.types.IMG} key={id} />
```

Comment:
Do we want to create (or change) a variation for that?

Reply:
I'm making sure with Meirav 👍